lib/api/objectHead.js

```diff
@@ -51,6 +52,8 @@ function objectHead(authInfo, request, log, callback) {
                     error: err,
                     method: 'objectHead',
                 });
+                monitoring.promMetrics(
+                        'HEAD', bucketName, err.code, 'headObject');
```

Comment:
don't you want to it for other error paths (line 61, 68, 74, 79)?

Reply:
Added.